front/app/containers/Admin/projects/project/phaseSetup/components/PhaseParticipationConfig/components/inputs/_shared/PrescreeningToggle.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import { Box, Toggle, Tooltip, Text } from '@citizenlab/cl2-component-library';
+
+import useFeatureFlag from 'hooks/useFeatureFlag';
+
+import { SectionField, SubSectionTitle } from 'components/admin/Section';
+
+import { FormattedMessage } from 'utils/cl-intl';
+
+import messages from '../../../../../../messages';
+
+interface Props {
+  prescreening_enabled: boolean | null | undefined;
+  togglePrescreeningEnabled: (prescreening_enabled: boolean) => void;
+}
+
+const PrescreeningToggle = ({
+  prescreening_enabled,
+  togglePrescreeningEnabled,
+}: Props) => {
+  const prescreeningEnabled = useFeatureFlag({ name: 'prescreening' });
+
+  return (
+    <SectionField>
+      <SubSectionTitle style={{ marginBottom: '0px' }}>
+        <FormattedMessage {...messages.participationOptions} />
```

Comment:
In the future, we might want to add more fields in this participation options section so I would not abstract it as part of the pre-screening toggle. Not critical, though.

Reply:
Indeed, because of the copy, I assumed other fields can be included in the section but if it's specifically about pre-screening, it would make sense to include it in the abstracted component.